### PR TITLE
feat(export): Export optimization

### LINKED ISSUE #376: 导出excel优化
### Description

导出excel优化
image

### Validations

- [ ] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by codejeffrey**
Export optimization
closed #376

### CODE REVIEW

File: packages/api-server/src/services/ProjectDetailService.ts [hunk 1]
```diff
@@ -344,13 +346,72 @@ export async function getSoftwareActivity(repoName: string): Promise<EcologyActi
   };
 }
 
-export async function exportScoreExcel(packageName: string) {
+export async function exportScoreExcel(projectName: string) {
   const excelTemplate = readFileSync('./assets/evaluation-template.xlsx');
-  const data = await EvaluationSummary.findOne({
+  const projectId = await getProjectIdByRepoName(projectName);
```
Comment (flyfishzy) [inline packages/api-server/src/services/ProjectDetailService.ts]: 是否能复用对比功能接口？

Comment (codejeffrey) [reply packages/api-server/src/services/ProjectDetailService.ts]: done

File: packages/api-server/src/services/ProjectDetailService.ts [hunk 2]
```diff
@@ -344,13 +346,27 @@ export async function getSoftwareActivity(repoName: string): Promise<EcologyActi
   };
 }
 
-export async function exportScoreExcel(packageName: string) {
+export async function exportScoreExcel(projectName: string) {
   const excelTemplate = readFileSync('./assets/evaluation-template.xlsx');
-  const data = await EvaluationSummary.findOne({
+  const res = await getProjectDetailInfo(projectName);
+  const data = Object(res);
```
Comment (flyfishzy) [inline packages/api-server/src/services/ProjectDetailService.ts]: 添加为ts可选类型

Comment (codejeffrey) [reply packages/api-server/src/services/ProjectDetailService.ts]: done

**Pull request merged**